api/src/main/java/org/openmrs/api/storage/LocalStorageService.java

```diff
@@ -59,7 +59,7 @@ public class LocalStorageService extends BaseStorageService implements StorageSe
 	
 	private final MimetypesFileTypeMap mimetypes = new MimetypesFileTypeMap();
 	
-	public LocalStorageService(@Value("${storage_dir:}") String storageDir, @Autowired StreamDataService streamService) {
+	public LocalStorageService(@Value("${storage.dir:}") String storageDir, @Autowired StreamDataService streamService) {
```

Comment:
Better to use . as a separator as it is more nicely structured in yaml/json files.